utilities/cleanSearch.js

```diff
@@ -0,0 +1,45 @@
+const checkForAllSpaces = (trend) => {
+  for (let i = 0; i < trend.length; i++) {
+    if (trend[i] !== ' ') {
+      return false;
+    }
+  }
+
+  return true;
+};
+
+const prepForAylien = (trend) => {
+  trend = trend.toLowerCase();
+
+  if (trend.length === 0 || checkForAllSpaces(trend)) {
```

Comment:
u prob already know this but another option for `if` `trend === null` and `trend.length === 0` is `!trend` and `!trend.length` 😃